app/presenters/tree_builder_containers.rb

```diff
@@ -16,8 +17,31 @@ def set_locals_for_render
     )
   end
 
-  # Get root nodes count/array for explorer tree
+  # level 0 - root
+  def root_options
+    [t = N_("All Containers (by Pods)"), t]
+  end
+
+  # level 1 - pods
   def x_get_tree_roots(count_only, _options)
-    count_only_or_objects(count_only, rbac_filtered_objects(Container.where(:deleted_on => nil)), "name")
+    list = []
+    objects = ContainerGroup.where(:deleted_on => nil).order(:name)
+    objects.each do |c|
```

Comment:
`list = objects.select ... .map` would look better maybe? but never mind, we can clean it later given this is an urgent fix